SQL/0000-00-00-schema.sql

```diff
@@ -2041,3 +2041,24 @@ CREATE TABLE `candidate_consent_history` (
   `EntryStaff` varchar(255) DEFAULT NULL,
   CONSTRAINT `PK_candidate_consent_history` PRIMARY KEY (`CandidateConsentHistoryID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
```

Comment:
missing Drop tables at top of document. Keep in ind the order of creation and dropping is important !!!

Reply:
resolved